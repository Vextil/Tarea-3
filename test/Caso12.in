# stress Binario
# Prueba de orden de tiempo
TiempoBinario
# Prueba de uso de memoria
MemoriaBinario
Fin
