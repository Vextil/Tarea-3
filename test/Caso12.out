1># stress Binario
2># Prueba de orden de tiempo
3>Fin Copia. Fin Linealizacion. Fin Balanceado. 
4># Prueba de uso de memoria
5>Fin Destruir. Fin Remover. Fin Balanceado. 
6>Fin
